exercises/square-root/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.
+
+As the first step in our calculation, we take a number and find its square root—that is, the number that, when multiplied by itself, equals the given number.
```

Comment:
```suggestion
As the first step in our calculation, we take to add functionality that can take a target number and find its square root — (i.e. the number that, when multiplied by itself, equals the target number).
```

I got a bit lost in the definition of the square root with the use of "number" to mean different numbers. It's correct, but a little hard to parse. So I think labelling the "target number" (better name welcome) makes it clearer. I'd also rather put the definition in brackets as, while I'm very :+1: on having it, I think most people know what a square root is, and it's potentially a little confusing having to read it.

Reply:
Is "take to add"  correct?  Seems to be a missing word in there.